### PR TITLE
Completed React Lists Exercise-Allyson Greven

### PR DESCRIPTION
**Pull request by allygreven**
Screenshot 2024-12-11 at 4 00 02 PM (2)

### CODE REVIEW

File: react-lists/my-app/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' },
+];
+
+export default function App() {
+  return (
+    <>
```
Comment (RobertGardner) [inline react-lists/my-app/src/App.tsx:14]: Unnecessary fragment!

File: react-lists/my-app/src/PokemonList.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+type Pokemon = {
+  number: string;
+  name: string;
+};
+
+export function PokemonList({ pokedex }: Props) {
+  return (
+    <>
```
Comment (RobertGardner) [inline react-lists/my-app/src/PokemonList.tsx:14]: This one, too.

**Review by RobertGardner — APPROVED** [review]

File: react-lists/my-app/src/PokemonList.tsx (hunk 2 above)
Comment (allygreven) [reply react-lists/my-app/src/PokemonList.tsx:14]: fixed!

File: react-lists/my-app/src/App.tsx (hunk 1 above)
Comment (allygreven) [reply react-lists/my-app/src/App.tsx:14]: fixed!

**Pull request merged**